google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -81,9 +83,17 @@
   private final FlowControlSettings flowControlSettings;
   private final boolean useLegacyFlowControl;
 
+  private AtomicBoolean enableExactlyOnceDelivery;
+
+  private final MessageReceiverWithAckResponse receiverWithAckResponse;
+
+  // Keeps track of requests without closed futures
+  private final Set<AckRequestData> pendingRequests = ConcurrentHashMap.newKeySet();
+
   private final AtomicLong channelReconnectBackoffMillis =
       new AtomicLong(INITIAL_CHANNEL_RECONNECT_BACKOFF.toMillis());
   private final Waiter ackOperationsWaiter = new Waiter();
+  private final ApiClock clock;
```

Comment:
I also don't think we need to store this, since it is only used in the constructor.